crowbar_framework/app/controllers/intelrsd_controller.rb

```diff
@@ -0,0 +1,245 @@
+#
+# Copyright 2016, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "json"
+
+# An important note on interfacing with Redfish APIS:
+#
+# Only specific top level URIs may be assumed, and even these
+# may be absent based on the implementation. (for ex: there
+# might be no /redfish/v1/Systems collection on something that
+# doesn't have compute nodes )
+#
+# The API will eventually be implemented on a system that breaks
+# any data model and hence the URIs must be dynamically discovered
+# The data model represented here using @node_object_list prepares
+# a list of all the available Systems along with the properties
+# and IDs of available resources in each system. This data model
+# needs to be appropriately mapped to the Node object of the system
+# where the data model is employed.
+#
+
+class RsdController < ApplicationController
+  attr_reader :logger, :insecure
+
+#  def initialize()
+#    @redfish_client = RedfishHelper::RedfishClient.new('localhost', '8443')
+#    @node_object_list = []
+#  end
+
+  def show
+    @title = "Welcome to RackScale Design"
+    @redfish_client = RedfishHelper::RedfishClient.new('10.160.66.119', '8443')
```

Comment:
Maybe we can initialise the client as part of the class so we dont have to create a client per method?